src/renderers/webgl/WebGLUniforms.js

```diff
@@ -53,7 +53,7 @@ import { CubeTexture } from '../../textures/CubeTexture.js';
 import { Texture } from '../../textures/Texture.js';
 
 var emptyTexture = new Texture();
-var emptyTexture3d = new Texture();
+var emptyTexture3d = new Texture3D();
```

Comment:
I think you have to include the following line, otherwise `Texture3D` is undefined.
```js
import { Texture3D } from '../../textures/Texture3D.js';
```